test/components/urlBarSuggestionsTest.js

```diff
@@ -150,11 +151,16 @@ describe('urlBarSuggestions', function () {
 
   it('selection is not reset when pressing non-input key', function * () {
```

Comment:
this test always fails when I run `npm run uitest -- --grep="urlBarSuggestions"`... but passes when it's by itself